src/core_plugins/console/public/index.html

```diff
@@ -1,38 +1,40 @@
-<sense-navbar></sense-navbar>
+<kbn-dev-tools-app>
+  <sense-navbar></sense-navbar>
```

Comment:
I would rather not have the stacking navbars caused by this, so I created https://github.com/elastic/kibana/issues/8234 to explore another option. It doesn't block this pr though.